player/lua/console.lua

```diff
@@ -691,28 +691,28 @@ function build_completers()
         }
     end
 
-    for _, command in pairs({'set', 'cycle[-_]values'}) do
+    for _, command in pairs({'set'}) do
```

Comment:
I think it's fine to remove this loop unless you have something specific in mind for which this can be used in the future?

Reply:
The one thing I had in mind was `change-list`, but I haven't looked at how hard it would be to implement that.

Reply:
I already implemented that in https://github.com/mpv-player/mpv/pull/12741

Reply:
I didn't know about that one, looking forward to it getting merged :)
I'll have a closer look at that later.